qemu/tests/hugepage_specify_node.py

```diff
@@ -44,6 +48,10 @@ def run(test, params, env):
         if node_memfree < mem_size:
             node_list.remove(node_id)
 
+    if 'arm64' in params['machine_type']:
```

Comment:
This should be a global checkpoint for all architectures?

Reply:
Nope, other platforms have different sizes on rhel8 and there is no check requirement.

Reply:
So I suggest keeping arm only

Reply:
`getconf PAGESIZE'` is a public command on all platforms, so this can be applied to all of them, instead of thinking about whether there is a requirement.

Reply:
@huangyum @MiriamDeng @fbq815 Hello, Does your platform need to join this checkpoint? If necessary, please add the corresponding version of PAGESIZE to bz. thanks.

Reply:
@zhenyzha Could you pls explain why add this check into this hugepage case? How is it related and why do we need this check?

Reply:
@zhenyzha on s390x, we have 1024kB for huge page size by default, and btw we only have one node on s390x

Reply:
> @zhenyzha Could you pls explain why add this check into this hugepage case? How is it related and why do we need this check?

Due to the code change patch and bz on arm.

Reply:
On ppc64le, we have 2M and 1G huge page size. 'getconf PAGESIZE' only helps to check the default page size on each architecture. I have same concern to yumei. Thanks.

Reply:
hello guys Chance for some security reasons, I will delete this comment. Let's discuss in bz.